lib/lightning/credentials.ex

```diff
@@ -19,6 +18,7 @@ defmodule Lightning.Credentials do
   alias Lightning.Credentials.SchemaDocument
   alias Lightning.Credentials.SensitiveValues
   alias Lightning.Projects.Project
+  # alias Lightning.Projects.ProjectCredential
```

Comment:
Remove the unused/commented out aliases please

Reply:
Done